device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/DeviceClient.java

```diff
@@ -41,6 +41,7 @@
  * </p>
  * The client supports HTTPS 1.1 and AMQPS 1.0 transports.
  */
+@SuppressWarnings("deprecation") // A lot of references to deprecated APIs, but all of them are still in use
```

Comment:
Moving this suppression to the class level because we have 10+ references to the now deprecated TransportClient and other deprecated fields

Reply:
I think this is alright. Will this suppress when we use other APIs that are depreciated from 3rd parties?